code/controllers/subsystem/hijack.dm

```diff
@@ -0,0 +1,371 @@
+
+#define EVACUATION_TYPE_NONE 0
+#define EVACUATION_TYPE_ADDITIVE 1
+#define EVACUATION_TYPE_MULTIPLICATIVE 2
+
+#define HIJACK_ANNOUNCE "ARES Emergency Procedures"
+#define XENO_HIJACK_ANNOUNCE "You sense something unusual..."
+
+#define EVACUATION_STATUS_NOT_INITIATED 0
+#define EVACUATION_STATUS_INITIATED 1
+
+#define HIJACK_OBJECTIVES_NOT_STARTED 0
+#define HIJACK_OBJECTIVES_STARTED 1
+#define HIJACK_OBJECTIVES_COMPLETE 2
+
+SUBSYSTEM_DEF(hijack)
```

Comment:
missing init_order
since /area/almayer registers to the subsystem you probably want this to init before mapload to avoid potential mistakes down the line